board.rb

```diff
@@ -0,0 +1,16 @@
+class Board
+
+  def initialize(level:)
+    @level = level
+  end
+
+  def to_s
+    lines = File.readlines('sokoban_levels.txt')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.